rad/src/operators/integer.rs

```diff
@@ -0,0 +1,245 @@
+    let negative_integer = RadonInteger::from(-10);
+
+    assert_eq!(absolute(&positive_integer).unwrap(), positive_integer);
+    assert_eq!(absolute(&negative_integer).unwrap(), positive_integer);
+    assert_eq!(
+        absolute(&RadonInteger::from(i128::min_value()))
+            .unwrap_err()
+            .to_string(),
+        "Overflow error".to_string(),
+    );
+}
+
+#[test]
+fn test_integer_to_float() {
+    let rad_int = RadonInteger::from(10);
+    let rad_float = RadonFloat::from(10.0);
+
+    assert_eq!(to_float(rad_int).unwrap(), rad_float);
+}
+
+#[test]
+fn test_integer_to_string() {
+    let rad_int = RadonInteger::from(10);
+    let rad_string: RadonString = RadonString::from("10");
+
+    assert_eq!(to_string(rad_int).unwrap(), rad_string);
+}
+
+#[test]
+fn test_integer_multiply() {
+    let rad_int = RadonInteger::from(10);
+    let value = Value::Integer(3);
+
+    assert_eq!(
+        multiply(&rad_int, &[value]).unwrap(),
+        RadonInteger::from(30)
+    );
+
+    let value = Value::Integer(3);
+    assert_eq!(
+        multiply(&RadonInteger::from(i128::max_value()), &[value])
+            .unwrap_err()
+            .to_string(),
+        "Overflow error".to_string(),
+    );
+}
+
+#[test]
+fn test_integer_greater() {
+    let rad_int = RadonInteger::from(10);
+    let value = Value::Integer(9);
+    let value2 = Value::Integer(10);
+    let value3 = Value::Integer(11);
+
+    assert_eq!(
+        greater_than(&rad_int, &[value]).unwrap(),
+        RadonBoolean::from(true)
+    );
+    assert_eq!(
+        greater_than(&rad_int, &[value2]).unwrap(),
+        RadonBoolean::from(false)
+    );
+    assert_eq!(
+        greater_than(&rad_int, &[value3]).unwrap(),
+        RadonBoolean::from(false)
+    );
+}
+
+#[test]
+fn test_integer_less() {
+    let rad_int = RadonInteger::from(10);
+    let value = Value::Integer(9);
+    let value2 = Value::Integer(10);
+    let value3 = Value::Integer(11);
+
+    assert_eq!(
+        less_than(&rad_int, &[value]).unwrap(),
+        RadonBoolean::from(false)
+    );
+    assert_eq!(
+        less_than(&rad_int, &[value2]).unwrap(),
+        RadonBoolean::from(false)
+    );
+    assert_eq!(
+        less_than(&rad_int, &[value3]).unwrap(),
+        RadonBoolean::from(true)
+    );
+}
+
+#[test]
+fn test_integer_negate() {
+    let positive_integer = RadonInteger::from(10);
+    let negative_integer = RadonInteger::from(-10);
+
+    assert_eq!(negate(&positive_integer).unwrap(), negative_integer);
+    assert_eq!(negate(&negative_integer).unwrap(), positive_integer);
+
+    assert_eq!(
+        negate(&RadonInteger::from(i128::min_value()))
+            .unwrap_err()
+            .to_string(),
+        "Overflow error".to_string(),
+    );
+}
+
+#[test]
+fn test_integer_modulo() {
+    let rad_int = RadonInteger::from(10);
+    let value = Value::Integer(3);
+
+    assert_eq!(modulo(&rad_int, &[value]).unwrap(), RadonInteger::from(1));
```

Comment:
Same as with floats, could you add more examples with negative numbers? Here is a nice discussion with examples: https://internals.rust-lang.org/t/mathematical-modulo-operator/5952